BraintreeCore/src/main/java/com/braintreepayments/api/core/DeviceInspector.kt

```diff
@@ -54,10 +56,16 @@ internal class DeviceInspector(
         return isVenmoIntentAvailable && isVenmoSignatureValid
     }
 
+    /**
+     * @suppress
+     */
```

Comment:
A single `@suppress` annotation at the `class DeviceInspector` level may be enough to keep the docs tool from generating docs for this class.

Reply:
😲 Updated: [b3f9a33](https://github.com/braintree/braintree_android/pull/1110/commits/b3f9a33b2d2f38491f68c1c45d717e1b30cbb0e5)